src/main/java/cr0s/warpdrive/config/Dictionary.java

```diff
@@ -270,6 +270,14 @@ public static void loadConfig(@Nonnull final Configuration config) {
 			config.get("block_tags", "ic2:blockchambers"                               , "PlaceLatest").getString(); // IC2 Classic Reactor chamber
 			config.get("block_tags", "immersiveengineering:connector"                  , "PlaceLatest").getString();
 			config.get("block_tags", "industrialforegoing:black_hole_label"            , "PlaceLatest").getString();
+			config.get("block_tags", "jsg:dhd_block"                                   , "PlaceLatest").getString();
+			config.get("block_tags", "jsg:dhd_pegasus_block"                           , "PlaceLatest").getString();
+			config.get("block_tags", "jsg:stargate_milkyway_base_block"                , "PlaceEarliest").getString();
```

Comment:
PlaceNormal is the default, is there any reasons to span the blocks so far away from each others? don't we corrupt the original structure during transfer here?

Reply:
I was using the SGCraft compatibility as a reference and this was the first fix I tried, to jump the blocks in the same order as SGCraft. It didn't seem to have any impact however, this does not cause any issues either. This may be able to be removed however I haven't tested that.